### PR TITLE
Users/arjgupta/deprecate task

### PR DESCRIPTION
**Pull request by arjgupta**

### CODE REVIEW

File: Tasks/AzureWebPowerShellDeployment/task.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        "Release"
+    ],
+    "runsOn": [
+        "Agent"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 46
```
Comment (kmkumaran) [inline Tasks/AzureWebPowerShellDeployment/task.json]: I assume the patch version was bumped up.

File: Tasks/IISWebAppDeployment/task.json [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 23
```
Comment (kmkumaran) [inline Tasks/IISWebAppDeployment/task.json]: Please confirm if the patch version was updated.

File: Tasks/SqlServerDacpacDeployment/task.json [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 18
```
Comment (kmkumaran) [inline Tasks/SqlServerDacpacDeployment/task.json]: Please confirm if the patch version was updated.

File: Tasks/AzureWebPowerShellDeployment/task.json (hunk 1 above)
Comment (arjgupta) [reply Tasks/AzureWebPowerShellDeployment/task.json]: yes

File: Tasks/IISWebAppDeployment/task.json (hunk 2 above)
Comment (arjgupta) [reply Tasks/IISWebAppDeployment/task.json]: yes, it was bumped up

File: Tasks/SqlServerDacpacDeployment/task.json (hunk 3 above)
Comment (arjgupta) [reply Tasks/SqlServerDacpacDeployment/task.json]: yes, it was bumped up

**Pull request merged**